src/index.js

```diff
@@ -19,22 +21,30 @@ const renderListOfStateOption = () => {
 
 const render = list => {
   const phoneHTML = (phone, district) => `
-  <p>Send Text: 
-    <a href="sms:${addCountryCode(phone)}
-     ?body=${body(district)}">${addCountryCode(phone)}
-    </a>
-  </p>
   <p>Call: 
    <a href="tel:${addCountryCode(phone)}">${phone}</a>
   </p>
+  <p>Send Text: 
+  <a href="sms:${addCountryCode(phone)}
+   ?body=${body(district)}">${addCountryCode(phone)}
+  </a>
+</p>
+<p>Send WhatsApp: 
+<a href="https://wa.me/${addCountryCodeWithoutPlus(phone)}?text=${body(
```

Comment:
```suggestion
<a href="https://wa.me/${addCountryCode(phone, '234')}?text=${body(
```

I think this should work.